File: app/src/main/java/com/ztn/recyclerviewdemo/gamerecyclerview/GameBean.java
package com.ztn.recyclerviewdemo.gamerecyclerview;

import java.util.List;

/**
 * Created by 冒险者ztn on 2018/1/24.
 * 游戏中被选中的黑格子
 */

public class GameBean {

    public List<checkedList> game;

    public static class checkedList {
        public List<Boolean> checked;
    }
}

File: app/src/main/java/com/ztn/recyclerviewdemo/game/GameBean.kt
package com.ztn.recyclerviewdemo.game

/**
 * Created by 冒险者ztn on 2019/3/7.
 * 游戏各自 bean
 * 1：自己。2：敌人。3：是否碰撞
 * 说实话，第三个参数没啥用，但是懒得改了
 */
data class GameBean(var gz: Triple<Boolean, Boolean, Boolean>)